LICENSE

```diff
@@ -19,3 +19,5 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+YEAR: 2019
```

Comment:
Don't put this in the PR. when you actually run the script/command to make the tarball, you want this file to only have the 2 lines you put in this file. It's weird, I know

Reply:
lol ok